database/seeders/CurriculosTableSeeder.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Curriculo;
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+
+class CurriculosTableSeeder extends Seeder
+{
+    private  $arrayCurriculos = [
```

Comment:
Aqui te faltaria poner un static
O en el foreach cambiar el "self::$arrayCurriculos" por un "$this->arrayCurriculos"